### PR TITLE
Lobby UI overhaul

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request
Basically just upgrades the lobby to use HTML and butons. Less chat spam too.

Inspired by TauCeti! https://github.com/TauCetiStation/TauCetiClassic/pull/6901

## Changelog
:cl:
add: Frontend UI has been overhauled.
/:cl:

### CODE REVIEW

**Comment by Cheshify** [comment]
Looks pretty neat, sure

**Comment by Gandalf2k15** [comment]
LZ669 is causing memory issues, so this PR also disables that.

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -7,7 +7,6 @@
 #ifndef LOWMEMORYMODE
 	#ifdef ALL_MAPS
 		#include "map_files\Mining\Lavaland.dmm"
-		#include "skyrat\planets\rockplanet.dmm"
```
Comment (KasparoVy) [inline _maps/_basemap.dm:10]: Fuuuuck x_x aww
Byebye rocky, you were my favourite while you lasted

**Comment by FlamingLily** [comment]
So when are you going to post a PR that's just
Rebases skyrat to be a tau ceti downstream

I mean, I certainly wouldn't be *against* that.....

**Comment by Gandalf2k15** [comment]
> So when are you going to post a PR that's just
> Rebases skyrat to be a tau ceti downstream
> 
> I mean, I certainly wouldn't be _against_ that.....

I don't want to work with old bay code, it's better this way.

**Comment by FlamingLily** [comment]
> 
> 
> > So when are you going to post a PR that's just
> > Rebases skyrat to be a tau ceti downstream
> > I mean, I certainly wouldn't be _against_ that.....
> 
> I don't want to work with old bay code, it's better this way.

Fair enough, I just find it funny how hard you're goin' on Tau Ceti features recently (not that I blame you god their shit is sexy)

**Comment by GoldenAlpharex** [comment]
So, we're test-merging it, and I think it's really neat, only thing is, you constantly have to click on the chat tab to use shortcuts for stuff like OOC, which can be a bit annoying. Other than that, could it be possible to have to be, like, on the buttons to click them? It makes me anxious that I can toggle them at any X position on the screen and that only the Y you're at matters, because I feel like I'll click something on accident.

That being said, it's really neat, I like it :)

**Comment by Gandalf2k15** [comment]
> So, we're test-merging it, and I think it's really neat, only thing is, you constantly have to click on the chat tab to use shortcuts for stuff like OOC, which can be a bit annoying. Other than that, could it be possible to have to be, like, on the buttons to click them? It makes me anxious that I can toggle them at any X position on the screen and that only the Y you're at matters, because I feel like I'll click something on accident.
> 
> That being said, it's really neat, I like it :)

The issue with these becomes very technical unfortunately. The way byond and winset works is pretty complicated. Not sure how to go about making it use hotkeys

**Comment by GoldenAlpharex** [comment]
> > So, we're test-merging it, and I think it's really neat, only thing is, you constantly have to click on the chat tab to use shortcuts for stuff like OOC, which can be a bit annoying. Other than that, could it be possible to have to be, like, on the buttons to click them? It makes me anxious that I can toggle them at any X position on the screen and that only the Y you're at matters, because I feel like I'll click something on accident.
> > That being said, it's really neat, I like it :)
> 
> The issue with these becomes very technical unfortunately. The way byond and winset works is pretty complicated. Not sure how to go about making it use hotkeys

Oh, that wasn't exactly what I meant, it was more along the lines of, I can't use hotkeys like O for OOC when I click on the "game" window, when you normally can. It's kind of like when TGUI is opened and focused.

**Comment by Gandalf2k15** [comment]
FMing this now, we'll also permanently remove LZ669 soon.

File: _maps/_basemap.dm (hunk 1 above)
Comment (KasparoVy) [reply _maps/_basemap.dm:10]: Sincerely hope it comes back some day

**Comment by KasparoVy** [comment]
> 
> 
> FMing this now, we'll also permanently remove LZ669 soon.

Fuuuck T_T bye-bye Big Rock

**Pull request merged**